clarifai_model_utils/llm_eval/main.py

```diff
@@ -318,11 +318,11 @@ def evaluate(self,
           generate_qa=generate_qa)
     elif isinstance(dataset, pd.DataFrame):  # local data
       df = dataset
-      assert dataset_id or app_id, ValueError(
-          f"`dataset_id` or `app_id` is empty when using local dataset. Please pass them to kwargs"
-      )
+      #assert dataset_id or app_id, ValueError(
+      #    f"`dataset_id` or `app_id` is empty when using local dataset. Please pass them to kwargs"
+      #)
     elif isinstance(dataset, HFDataset):
-      df = dataset.to_pandas(batched=True)
+      df = dataset.to_pandas(batched=False)
```

Comment:
Why?

Reply:
When `batched=True`, it returns a generator, which will ultimately be converted to a DataFrame for saving to disk.